### ISSUE TITLE
Minimize app to System Tray on Windows

### ISSUE THREAD

**Issue by shaunroselt** [shaunroselt/Roselt-Developer-Tools#19]
It would be cool if one of the settings on Windows is to let the app minimize to the system tray instead of taskbar and have an icon there:
<img width="495" alt="image" src="https://user-images.githubusercontent.com/5418178/217075241-fb8d56d2-4086-40e0-bed8-417761bf4248.png">

I'm not sure about MacOS and Linux though, but I suppose it's okay if this is a Windows only feature?

Anyone, any ideas?

**Comment by gorhinstroebel** [comment]
Hi @shaunroselt, macos also has a "System Tray" its called the Control Center
![b094cb3e90f507b54f60475639233765](https://user-images.githubusercontent.com/102311306/217083907-af9ead1e-ceda-4735-9a9b-54cb34c081a7.png)

Hopefully this image from https://support.apple.com/en-za/guide/mac-help/mchlp1446/mac will help


Ill also attach a screenshot of apps that is currently running in my control center: 
![Screenshot 2023-02-06 at 22 58 33](https://user-images.githubusercontent.com/102311306/217084113-28666709-3923-40c5-a86e-563b1f55147f.png)

**Comment by shaunroselt** [comment]
> Hi @shaunroselt, macos also has a "System Tray" its called the Control Center ![b094cb3e90f507b54f60475639233765](https://user-images.githubusercontent.com/102311306/217083907-af9ead1e-ceda-4735-9a9b-54cb34c081a7.png)
> 
> Hopefully this image from https://support.apple.com/en-za/guide/mac-help/mchlp1446/mac will help
> 
> Ill also attach a screenshot of apps that is currently running in my control center: ![Screenshot 2023-02-06 at 22 58 33](https://user-images.githubusercontent.com/102311306/217084113-28666709-3923-40c5-a86e-563b1f55147f.png)

And any app can minimize or put their app there instead of the taskbar/dock ?

**Comment by shaunroselt** [comment]
@gorhinstroebel  And any app can minimize or put their app there instead of the taskbar/dock ?

**Comment by gorhinstroebel** [comment]
Yes, for example the application running on the far left (RDM) allows me to change the resolution of the screen

Thank you too the creator of this tool avibrazil here is the github if anyone want's to use the application. Its amazing.
https://github.com/avibrazil/RDM

**Comment by shaunroselt** [comment]
That's cool. Thanks @gorhinstroebel. MacOS should also then be supported with this feature.
